make/tools.mk

```diff
@@ -33,6 +34,8 @@ K8S_CODEGEN_VERSION=v0.24.2
 
 KUBEBUILDER_ASSETS_VERSION=1.22.0
 
+VENDORED_GO_VERSION := 1.18.3
+
```

Comment:
note: unrelated change; moved this up here because I like having all the versions together at the top